apps/src/templates/sectionProgressV2/coming-soon-banner.module.scss

```diff
@@ -26,6 +28,7 @@
   }
 
   & .bannerText {
+    flex: 1 1 80%;
```

Comment:
```suggestion
    flex-grow: 1;
```
Do we need to set flex-shrink or flex-basis or can we just use flex-grow?

Reply:
Thanks! Tested this change locally 👍